posts/2025-10-23-mcp-standalone-blog.adoc

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Integrate your Liberty app with agentic AI using MCP server"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/habiblawal1
+author_github: https://github.com/habiblawal1
+seo-title: Support for the Model Context Protocol within Open Liberty - OpenLiberty.io
+seo-description: Learn how to use the MCP Server feature within your Liberty application to allow your business logic to be utilized within an agentic AI workflow.
+blog_description: Learn how to use the MCP Server feature within your Liberty application to allow your business logic to be utilized within an agentic AI workflow.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Integrate your Liberty app with agentic AI using MCP server
+Habib Lawal <https://github.com/habiblawal1>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+== What is MCP
+Model Context Protocol (MCP) is an open standard that enables AI applications to interact with and utilize external systems.
+
+The Liberty **MCP Server feature** allows developers to expose the business logic of their applications, making it discoverable, understandable, and usable by AI applications.
+
+
+== The Power of MCP
+MCP has emerged as the standard for AI applications to access real-time information from external sources. This approach delivers more accurate and timely responses to users without the constant need to retrain AI models on new information.
+
+Consider a scenario where your company provides weather forecasting services that require continuous updates from multiple data sources. It's impractical and inefficient to constantly retrain your AI model with these frequently changing weather forecasts.
+
+A more effective solution is to enable the AI to access current weather data through tools exposed by your Liberty application. This allows the AI to retrieve up-to-date forecast information whenever needed, ensuring responses are always based on the most current data available, without the need for AI model retraining.
+
+== How to Use the Liberty MCP Server Feature
+The Liberty MCP Server feature enables a Liberty server to communicate with agentic AI workflows using the MCP protocol with https://modelcontextprotocol.io/specification/2025-03-26/basic/transports#streamable-http[streamable HTTP]. The feature `mcpServer-1.0` conforms to version https://modelcontextprotocol.io/specification/2025-06-18[2025-06-18] of the MCP protocol.
```

Comment:
Putting this here makes it sound like we support all of MCP 2025-06-18. It's not until the "Supported MCP Functionality" section that we say we only support tools.

I think this statement about the version of MCP that we support would be better under the "Supported MCP Functionality" section.

In addition, we do also support 2025-03-26 (the differences are minimal)